templates/opt/superset/superset_config.py.erb

```diff
@@ -17,6 +17,7 @@ SQLALCHEMY_DATABASE_URI = 'postgresql://<%= @db_user %>:<%= @db_pass %>@<%= @db_
 ROW_LIMIT = <%= @row_limit %>
 
 FEATURE_FLAGS = {
+    "DYNAMIC_PLUGINS": <%= @dynamic_plugins %>,
```

Comment:
Change to:
`    "DYNAMIC_PLUGINS": <% if @dynamic_plugins -%>True<% else -%>False<% end -%>`
to reflect actual boolean type, rather than string